fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -1144,6 +1150,14 @@ public void setMaxJoinNumberOfReorder(int maxJoinNumberOfReorder) {
     @VariableMgr.VarAttr(name = ENABLE_MEMTABLE_ON_SINK_NODE, needForward = true)
     public boolean enableMemtableOnSinkNode = false;
 
+    @VariableMgr.VarAttr(name = FULL_AUTO_ANALYZE_START_TIME, needForward = true)
+    public String fullAutoAnalyzeStartTime = "";
+
+    @VariableMgr.VarAttr(name = FULL_AUTO_ANALYZE_END_TIME, needForward = true)
+    public String fullAutoAnalyzeEndTime = "";
```

Comment:
should use global variable, not session variable

Reply:
It's hard to implement over current framework